### PR TITLE
fix typos

### PR DESCRIPTION
**Pull request by nully0x**
This PR fix some typo in the codebase.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -62,6 +62,7 @@ To contribute to Webhook Handler, follow these steps:
 3. Make your changes and commit them: `git commit -m '<commit_message>'`
 4. Push to the original branch: `git push origin <project_name>/<location>`
 5. Create the pull request.
+<<<<<<< Updated upstream
```
Comment (daviroo) [inline README.md]: Can you remove the git diff?

File: README.md [hunk 2]
```diff
@@ -99,3 +100,5 @@ To build and run the project using Docker for local development:
 Replace `<container_id>` with the actual container ID, which you can get by running `docker ps`.
 
 Note: For hot-reloading in development, you may need to adjust the Dockerfile and use a tool like nodemon.
+=======
+>>>>>>> Stashed changes
```
Comment (daviroo) [inline README.md]: Same here.

**Review by daviroo — CHANGES_REQUESTED** [review]
Some hanging git diff syntax.

File: README.md (hunk 2 above)
Comment (nully0x) [reply README.md]: Oops, I removed it, I guess my editor play error on me.

**Comment by nully0x** [comment]
@daviroo this is resolved

File: README.md [hunk 3]
```diff
@@ -47,7 +47,7 @@ The webhook handler exposes a single endpoint:
 
 - POST `/webhook`
 
-To test the webhook handler with the git-servie:
+To test the webhook handler with the git-service:
 
 1. Head on to [hxckr-infra]() and setup git-service which also starts soft-serve.
```
Comment (Extheoisah) [inline README.md]:
```suggestion
1. Head on to [hxckr-infra](https://github.com/extheoisah/hxckr-infra) and setup git-service which also starts soft-serve.
```

**Review by Extheoisah — APPROVED** [review]

**Review by daviroo — APPROVED** [review]

**Pull request merged**